scss/general.scss

```diff
@@ -5,6 +5,11 @@
 	box-sizing: border-box;
 }
 
+a[href^="http://"],
+a[href^="https://"] {
+	cursor: pointer; // Make all web links use pointer cursor
+}
```

Comment:
Is using the patterns here an indication that we have failed to use `<a />` appropriately in our app? Is there a reason they shouldn't all use the pointer?